common/decisions/dvg_batavia_decisions.txt

```diff
@@ -0,0 +1,160 @@
+﻿dvg_form_batavia_ops_decision = { #opstalia
```

Comment:
Always check if countries exist before checking if they are subjects or want to access it. So possible should include all involved countries exist check, or skip effects if they do not exist anymore.